### ISSUE TITLE
British Empire and Canada as part of CANZUK

### ISSUE THREAD

**Issue by simsulla** [simsulla/nwomod#685]
**Symphonia,**
I was thinking if the UK player refused to withdraw from India in a hurried fashion and instead took a slower approach to decolonization, the Dominions could remain in a British-centered worldview. That way, Britain could theoretically still have an Imperial / Commonwealth role for any of the Dominions. It serves as an immediate alternative to the more US-oriented roles that the former Dominions played in the RL postwar era. The point being, that a British Empire that takes slower to decolonize may still have a chance at federating its core constituencies through decades of economic and social integration, and then releasing the colonies as states. Any part of the Commonwealth that falls out of the British sphere of influence can take an independent path that irrevocably ends any chance of them being a part of the federated  system.

For example, declining immediate Indian independence at the Direct Action Day event could theoretically preserve the British Raj. This stops India from leaving the Empire and may perhaps preserve the idea of Britain's postwar superpower status. Immediately afterwards, in a 1946 Commonwealth Conference, the issue of imperial governance could come up, with decisions to retain the current system of arrangements, further decentralize and decolonize the Commonwealth, or to pull the Commonwealth tighter together. Depending on the player's decision and ability to retain the UK's superpower status, this decision could lead to a federated Commonwealth decades down the line.

In the RL 1946 Commonwealth Conference, one of the first things that they stated was that the British Commonwealth's members preferred "that the methods practiced now are preferable to any rigid centralized machinery". The quote can be found HERE in the Commonwealth Secretariat's 1946 communique.

This mod trying to follow realism while allowing for some ahistorical decision making, a player could try to make a case for the opposite, kind of a "Lets try and see if this works" option. It would entail the UK having to make sure that the Commonwealth members remained in the British sphere, while trying to fend off American or Russian attempts to speed up British decolonization. Here, the usual game mechanics would work, especially in regards to decolonization. Any event that results in the UK being forced to release colonies would end the idea of Britain's superpower status, especially if America uses the 1 million wartime debts event and causes the UK to declare bankruptcy.

Of course, it's not going to be this easy. Part of American wartime policy was to dismantle the economic strength of the British Empire, and a revival of Britain's role isn't really going to be welcomed by the USA. Furthermore, it has always been American geopolitical strategy to make sure regional powers could never become global hegemonic powers, or even truly dominate their own region without American backing.This could be simulated as the UK having to pay more money to the USA as part of its wartime debts, as the USA could try to economically break the UK's global role. For example, the 1 million wartime debt event could be tripled to 3 million. Furthermore, the USA could take harsher stances towards a Commonwealth that is trying to deal with the Suez crisis or a Falklands conflict.

If the UK can carry out its mission to maintain the Commonwealth, then I think that the player can deserve some reward when the year 1970 arrives and they can fire the CANZUK event and begin the process of creating a federal Commonwealth.

I'm saying all this because breaking up Canada should become a decision when the actual Commonwealth country is formed, since it would be hard to reconcile Quebecois to what they would definitely see as a return of the monarchy and of the British. However, giving a backstory that could possibly reconcile the political difference with economic integration could somehow avert that.

EDIT: Some event chain for releasing the African and Asian colonies as independent states would definitely be needed. Probably one of the problems for trying to maintain British superpower status is heightened militancy in the colonies, raising the threat that America could use the 3 million wartime debt to ruin the federal Commonwealth project.[/QUOTE]

**Comment by simsulla** [comment]
**Moridin997**
This seems pretty good. I imagine there would be an alternative for the UK to refuse paying the USA if it meant bankruptcy. Then the USA would be forced to decide how far it was willing to go to dismantle the British colonial empire...[/QUOTE]

**simsulla**
Great historic analysis :) . There is a historic parallel to a more centralized Commonwealth, the French Union. It failed because nationalist leaders rejected the idea, and wanted full independence.  So the UK needs to deal with not only the external US threat and policy to dismantle the economic strength of the British Empire but also nationalist sentiments among its colonials.[/QUOTE]

**Symphonia**
True enough. If the player can preserve as much of the British Empire until 1970 without successful colonial independence in Africa, holding India, and avoid American-induced bankruptcy, then 30 years of continued British "strength" postwar could mean that a united Commonwealth core can be viable. Events like the Falklands War and the Suez Crisis can open avenues for which the USA and the USSR could try to derail the project.

Basically, I'm trying to find a way such that Canada (or any other Old Commonwealth member) doesn't become a full-out presidential democracy before 1970. If nationalism and regional loyalties are forces that drive political unions apart, then only economic integration can serve to unite them.  Britain has to survive external and internal political and economic threats for 30 or so years in order to prove that it still can compete for a global leadership role. Only then, could any British leader follow through on decolonization efforts as a viable political option, because only then has Britain has found a postwar role.

**Comment by simsulla** [comment]
This is a proposal for how to design this.
All members of the Commonwealth will have a Commonwealth cores. Having a Commonwealth core means that you are a committed member, so it would be a smaller subset of the real-world Commonwealth countries. 1946 UK, Canada, Australia, NZ, The Raj, South Africa and Rhodesia would have Commonwealth cores. 1992 none. As the UK gives colones independence they will be offered to join the Commonwealth. depending on militancy they will decide to do so or not. This would be a reason for the UK to be very inclusive in handling colonial decent. Every 20 year or so an event checks the status of Commonwealth members. If a country is part of the UK sphere they will stay in the Commonwealth. If no-longer part of it they will leave - irreversibly.  

The current "imperial federation" will be retired, and replaced with the Commonwealth. Around 2000 (1970 seems to early given the decolonization wave at that time) there would be an option for the UK to make vassals of its Commonwealth members (i.e part of free trade, movement and globalization; a tighter Commonwealth), and then 2050 become a centralized country the Commonwealth. 

To further make it challenging, Australia, New Zealand and Canada must be part of the inner circle of the Commonwealth (i.e. in UK sphere), or the "centrist" Commonwealth will fade. So it is an absolute must to sphere for the UK to sphere Canada once they drop put of GP status and do so before the mid 1960s when the event that checks the status of Commonwealth, will forever make them leave the path of tighter integration and the possibility of a centrist Commonwealth.  

The mechanism for the French Union will be the same.

**Comment by simsulla** [comment]
After second thought, My scenario is simply to unrealistic. And after a review of the current #251 implementation, we will keep the current Imperial Union / Commonwealth implementation. This is a duplicate of that issue. Closing it.

**Comment by simsulla** [comment]
I am struggling with the "play-ability" of UK, France (Netherlands, etc). Of-cause as a player you want to maintain your empire, but the success of that is slim, given the capacity of native populations grow together with the cost to suppress them.  The solution is to release your colonies as vassals and then keep them in your sphere and vassal ship. But accept you can't control the territory directly. A closer community requires some other undignified denominational - the existing #251 does that, as the super states. A colonial heritage with no common religion, culture does not match those.
